frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js

```diff
@@ -351,7 +317,7 @@ class DataCube {
           });
         }
 
-        if (this.buckets.length > (2 * constants.MAXIMUM_BUCKET_COUNT_PER_LAYER) / 10) {
+        if (this.buckets.length > 2 * constants.MAXIMUM_BUCKET_COUNT_PER_LAYER) {
```

Comment:
```suggestion
        if (this.buckets.length > 2 * this.BUCKET_COUNT_SOFT_LIMIT) {
```

Did you deliberately decide against this suggestion? I know the two are the same, but the documentation at the top of the file only talks about `BUCKET_COUNT_SOFT_LIMIT` and it is also used a few lines above. Using it here as well is less confusing in my opinion :)

Reply:
Oh, sorry, no, that wasn't my intention. I thought the suggestion only aimed to remove the `/ 10`. When I cleaned up the debugging code, I did this manually. Changed it now :)